.devcontainer/devcontainer.json

```diff
@@ -29,18 +29,16 @@
   "customizations": {
     "vscode": {
       "extensions": [
-        "github.vscode-pull-request-github",
-        "eamodio.gitlens",
         "dbaeumer.vscode-eslint",
         "esbenp.prettier-vscode",
-        "bierner.lit-html",
```

Comment:
Removed this lit-html extension as it seems abandoned and superseded by the one below.